analysis.cxx

```diff
@@ -56,18 +56,24 @@ int main(){
     // Build the pair !
     auto df24_1 = physicsobject::FilterObjects(df24, "nTau", 1,"NumberOfTaus");
     auto df24_2 = physicsobject::FilterObjects(df24_1, "nMuon", 1,"NumberOfMuons");
-    auto df25 = pairselection::mutau::PairSelection(df24_2, "good_taus_mask", "good_muons_mask", "mtpair", {""});
-    auto df_final = df25;
+    auto mt_df = pairselection::mutau::PairSelection(df24_2, "good_taus_mask", "good_muons_mask", "mtpair", {""});
+    auto mt_df_1 = pairselection::filterGoodParticles(mt_df, "mtpair", "GoodMuTauPairs");
```

Comment:
Suggest to rename to `filterGoodPair` for better understanding (please see also comment below in the `pairselection.cxx`)